tests/Backend/FileWorkspace/WorkspacesLoadTest.php

```diff
@@ -781,6 +804,36 @@ public function testSourceTableNotFound()
         }
     }
 
+    public function testDataFileIdNotFound()
```

Comment:
Chybí mi testík na to že ten file existuje ale token na něj nemá oprávnění. Obdoba je v `CommonFileTest::testFilesPermissionsCanReadAllFiles()`

Reply:
Test jsem doplnil https://github.com/keboola/storage-api-php-client/pull/529/commits/594bcf2abad8771fa9063d632c69efa4af0677e2